packages/flutter/lib/src/material/slider.dart

```diff
@@ -1472,11 +1474,24 @@ class _RenderSlider extends RenderBox with RelayoutWhenSystemFontsChangeMixin {
   void describeSemanticsConfiguration(SemanticsConfiguration config) {
     super.describeSemanticsConfiguration(config);
 
-    config.isSemanticBoundary = isInteractive;
+    // The Slider widget has its own Focus widget with semantics information,
+    // and we want that semantics node to collect the semantics information here
+    // so that it's all in the same node: otherwise Talkback sees that the node
+    // has focusable children, and it won't focus the Slider's Focus widget
+    // because it thinks the Focus widget's node doesn't have anything to say
+    // (which it doesn't, but this child does). Aggregating the semantic
+    // information into one node means that Talkback will recognize that it has
+    // something to say and focus it when it receives keyboard focus.
+    // (See https://github.com/flutter/flutter/issues/57038 for context).
+    config.isSemanticBoundary = false;
+
     if (isInteractive) {
       config.textDirection = textDirection;
       config.onIncrease = increaseAction;
       config.onDecrease = decreaseAction;
+      if (_label != null && _label.isNotEmpty) {
```

Comment:
Looking at this, is it correct that the label (and some of the other properties here) are only published when the slider is interactive? I'd assume a disabled slider should also have a label?

Reply:
Yeah, I thought that was weird too, but I didn't want to change it without understanding it. @clocksmith, is there a reason for that?